src/guides/v2.3/javascript-dev-guide/javascript/js_init.md

```diff
@@ -111,6 +111,20 @@ On DOM ready, the `data-mage-init` attribute is parsed to extract component name
   };
   ```
 
+Where `<component_name>` native Magento JS component, for example: `menu`, `collapsible`, `tooltip` ...
+
+```html
+<nav data-mage-init='{"tooltip": {"content": "<?= /* @noEscape */ $content ?>"}}'></nav>
+```
+
+Or a custom JS component, can be component path `Vendor_Module/js/component` or alias declareted in `requirejs-config.js`.
```

Comment:
```suggestion
Or a custom JS component, implemented with a component path: `Vendor_Module/js/component`, or as an alias declared in `requirejs-config.js`.
```
If this suggestion is not accurate, than some kind of grammar fix is needed here.